sql_wrappers.go

```diff
@@ -694,23 +694,23 @@ func get_conn_Execer_ExecerContext_Queryer_QueryerContext_ConnPrepareContext_Nam
 			sensor:      sensor,
 		},
 		Execer: &wExecer{
-			Execer:      Execer,
```

Comment:
@Angith We generate this file from tpl file in internal/bin/sql folder. Are we editing it manually?

Reply:
Hi @sanojsubran. I edited this file manually. It seems it should not be edited manually. Let me try to generate it conventionally.

Reply:
@sanojsubran I updated the `sql_wrappers.tpl` file to include the `sqlSpan` struct and generated the `sql_wrappers.go` using the updated template.